runtime/src/bank/fee_distribution.rs

```diff
@@ -80,13 +80,25 @@ impl Bank {
         fee_details: &CollectorFeeDetails,
     ) -> (u64, u64) {
         let (deposit, burn) = if fee_details.transaction_fee != 0 {
-            self.fee_rate_governor.burn(fee_details.transaction_fee)
+            self.burn_transaction_fee(fee_details.transaction_fee)
         } else {
             (0, 0)
         };
         (deposit.saturating_add(fee_details.priority_fee), burn)
     }
 
+    fn burn_transaction_fee(&self, fees: u64) -> (u64, u64) {
+        let burned = fees * u64::from(self.burn_percent()) / 100;
+        (fees - burned, burned)
+    }
+
+    fn burn_percent(&self) -> u8 {
```

Comment:
just return this as a u64?
also fn could be `const (though it's likely inlined as a const anyway)

Reply:
c0ebf47b297b465ec44c1a8d1d3c0ee21c4c2203
`u64::from` makes it non-const tho